src/xrc/src/exchanges.rs

```diff
@@ -114,6 +114,12 @@ macro_rules! exchanges {
                     $(Exchange::$name(exchange) => exchange.max_response_bytes()),*,
                 }
             }
+
+            /// This method returns whether the exchange should be called.
+            /// Currently, only IPv6 support determines whether is should be used.
```

Comment:
This comment is misleading because there is a check if IPv4 is available. It just happens that it always return `false` right now.
I think it's better to put such a comment together with the `is_ipv4_support_available` function so that the comment is updated together with the implementation.

Reply:
Makes sense. I have updated the comments.